ckanext/xloader/config_declaration.yaml

```diff
@@ -0,0 +1,116 @@
+version: 1
+groups:
+  - annotation: ckanext-xloader settings
+    options:
+      - key: ckanext.xloader.jobs_db.uri
+        example: sqlite:////tmp/xloader_jobs.db
+        description: |
+            The connection string for the jobs database used by XLoader. The
+            default of an sqlite file is fine for development. For production use a
+            Postgresql database.
+        validators: not_missing
+        required: true
+      - key: ckanext.xloader.api_token
+        example: eyJ0eXAiOiJKV1QiLCJh.eyJqdGkiOiJ0M2VNUFlQWFg0VU.8QgV8em4RA
+        description: |
+            Uses a specific API token for the xloader_submit action instead of the
+            apikey of the site_user. Will be mandatory when dropping support for
+            CKAN 2.9.
+        required: false
+      - key: ckanext.xloader.formats
+        example: csv application/csv xls application/vnd.ms-excel
+        description: |
+            The formats that are accepted. If the value of the resource.format is
+            anything else then it won't be 'xloadered' to DataStore (and will therefore
+            only be available to users in the form of the original download/link).
+            Case insensitive. Defaults are listed in plugin.py.
+        required: false
+      - key: ckanext.xloader.max_content_length
+        default: 1000000000
```

Comment:
I'd say that using `_` simplifies reading such values. `default: 1_000_000_000` is a valid value for the `int` type.